restapi/src/main/java/io/stargate/web/docsapi/service/ReactiveDocumentService.java

```diff
@@ -330,7 +372,10 @@ public ReactiveDocumentService(
       String documentId,
       List<String> subPath,
       String payload,
-      ExecutionContext context) {
+      String ttl,
```

Comment:
same here.. as we need to handle both update of the whole document with ttl int and the ttl auto I propse:

1. `updateDocument` accepts ttl int and calls `updateDocumentInternal`
2. add `updateSubDocument` with ttl-auto boolean and if it's `true` resolves the ttl and calls the `updateDocumentInternal` with the value.
3. this method is renamed to `updateDocumentInternal` and accepts only ttl as int